data-formats/src/ownershipvoucher.rs

```diff
@@ -441,6 +441,29 @@ impl OwnershipVoucherHeader {
     }
 }
 
+fn check_device_info(device_info: &str) -> Result<bool> {
+    let mut chars = device_info.chars();
+    let result = chars.all(|f| {
+        let acceptable_ascii_range = 32..127;
+        acceptable_ascii_range.contains(&(f as u8))
+    });
+    Ok(result)
```

Comment:
Let's change this function to `-> Result<()>`, and just have something like:
```
if result {
    Err(Error::InconsistentValue("Invalid values in Device Info"))
} else {
    Ok(())
}
```

Reply:
Done